### DISCUSSION TITLE
How to change repository description?

### DISCUSSION THREAD

**Discussion by korn3r** [community/community#60507] [New to GitHub] [answered]
Hi. I'm new to repositories.
During creation there was a window, optional one, but i decided to write a bunch of text there. I accidentally pressed enter and repo created with a small portion of description i wanted to give it.
How do i edit description of repo? I looked everywhere. There is a big bunch of settings i dont even understand meaning of, but none of that seems like "edit description" option.

And yes, i did used search and found this discussion with answer marked as right, but github looks slightly different since that screenshot has been made.
https://github.com/orgs/community/discussions/54372

**Comment by Gustavonn07** [comment, marked as answer]
If you're talking about creating a readme, create a README.md file and write whatever you want in the project, it will appear for everyone who accesses your repository.

If it's just the description, follow this step by step:

1. in your repository go to config following the arrow: 

<hr>

![image](https://github.com/community/community/assets/84361085/405541dc-0fe7-448d-8df6-21bf02fae9a0)

<hr>

2.  edit the description:

<hr>

![image](https://github.com/community/community/assets/84361085/4051b307-a2e2-4805-bf66-1a7ed2bb60b8)

<hr>

Hope it helps! If I helped you can mark my answer as the oficial to help other person with the same problem or just to say It worked!

**Reply by korn3r** [reply]
Did you tried it yourself?
there is no such icon anymore (atleast for me). There is settings tab, but it does not contain anything related to description.

**Reply by Gustavonn07** [reply]
hmmm this is strange, this step by step I did in my repository (Im brazilian thats why my repository is in portuguese), I use desktop on github, do you use mobile?

I was searching on github and it seems like last year there wasn't a way to edit description on mobile, I also tried looking but couldn't find anything.
I found this discussion on github: https://github.com/orgs/community/discussions/28097
and I realized using devtools that the tab appears on devices with width bigger than 767px.

edit: it seems that there is a way to create descriptions when creating the repository for the 1st time (the only way I found to create a description on mobile)

**Reply by aflacake** [reply]
**Under itself, above package or deploy.** The gear icon is very small so it is hard to find for new users. Previously, I have posted this issue in [bug].

**Comment by mbsaerens** [comment]
For those who cannot find the 'About' panel: Your project needs to have a branch first.
I didn't see this panel either when I had just created an empty repository. Once I made a first commit on the main branch, this panel showed up and I could modify the description.

**Comment by Elanrif** [comment]
Great !

**Comment by scg90** [comment]
THANK YOU SO MUCH UR A LIFESAVER

**Comment by mubtashimrafid** [comment]
You can edit the repository description pretty easily — GitHub just hides it a bit for new users.

1. Open your repository.
2. Go to the main **Code** page of the repo.
3. On the right side, look for the **About** section (near the top-right).
4. Click the small gear icon or pencil icon.
5. Edit the description and save.

You can also do it through **Settings → General → Repository details** and update the description there.

If your repository is completely empty, GitHub may not show the description editor yet. In that case, add a file first (like a README), then the edit option should appear. ([[github.com](https://github.com/orgs/community/discussions/170808?utm_source=chatgpt.com)][1])

[1]: https://github.com/orgs/community/discussions/170808?utm_source=chatgpt.com "How to Update the Description of Your Repository on GitHub · community · Discussion #170808 · GitHub"
